integrate/data-streams/overview/index.md

```diff
@@ -54,45 +71,45 @@ The `template` column type currently only supports the `jsonnet` templating lang
 
 ### Conditions
 
-Incoming events are immediately mapped to the schema defined in each export, and each new row is appended to a CSV file. This CSV file is stored in your Keboola project. When certain conditions are met, the data from the file is uploaded to the destination table, and the file is cleared. These `conditions` are defined by the export:
+Incoming events are immediately mapped to the schema defined in each sink, and each new row is appended to a CSV file. This CSV file is stored in your Keboola project. When certain conditions are met, the data from the file is uploaded to the destination table, and the file is cleared. These `conditions` are defined by the sink:
 
 | Condition | Minimum | Maximum | Default |
 | :- | :-: | :-: | :-: |
 | `time` | 30 seconds | 24 hours | 5 minutes |
 | `size` | 100 B | 50 MB | 5 MB |
 | `count` | 1 | 10 million | 1 thousand |
```

Comment:
I think defaults have changed. Make sure with Tomis that we have current defaults up to date.